data/sql/updates/pending_db_world/Fix_quest_11879.sql

```diff
@@ -0,0 +1,60 @@
+(25802, 1, 0, 'Kaw\'s halberd breaks in two and falls to the ground!', 41, 0, 100, 0, 0, 930, 25087, 0, 'Kaw the Mammoth Destroyer Death');
+-- Kaw SAI
+DELETE FROM `smart_scripts` WHERE (`source_type` = 0 AND `entryorguid` = 25802);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `event_param6`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(25802, 0, 0, 0, 1, 0, 100, 1, 0, 0, 0, 0, 0, 0, 18, 33024, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Kaw the Mammoth Destroyer - Out of Combat - Set Flags Immune To Players (No Repeat)'),
+(25802, 0, 1, 0, 101, 0, 100, 1, 1, 20, 0, 0, 0, 0, 80, 2580200, 2, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Kaw the Mammoth Destroyer - On Min Players in Range - Run Script (No Repeat)'),
+(25802, 0, 2, 0, 6, 0, 100, 0, 0, 0, 0, 0, 0, 0, 11, 46310, 2, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Kaw the Mammoth Destroyer - On Just Died - Cast \'Drop War Halberd\''),
+(25802, 0, 3, 0, 6, 0, 100, 1, 0, 0, 0, 0, 0, 0, 1, 1, 1000, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Kaw the Mammoth Destroyer - On just died - Alert Message');
+-- Kaw events
+DELETE FROM `smart_scripts` WHERE (`entryorguid` = 2580200) AND (`source_type` = 9) AND (`id` IN (0, 1, 2, 3, 4));
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(2580200, 9, 0, 0, 0, 0, 100, 1, 0, 0, 0, 0, 0, 1, 0, 2000, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Kaw - Timed - Yell'),
+(2580200, 9, 1, 0, 0, 0, 100, 1, 0, 0, 0, 0, 0, 59, 1, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Kaw - Timed - Enable Running'),
+(2580200, 9, 2, 0, 0, 0, 100, 1, 3000, 3000, 0, 0, 0, 69, 0, 0, 0, 0, 0, 0, 8, 0, 0, 0, 0, 3974.17, 5476.31, 35.602, 5.564, 'Kaw - Timed - Move to Moria'),
+(2580200, 9, 3, 0, 0, 0, 100, 1, 2500, 2500, 0, 0, 0, 11, 46260, 2, 0, 0, 0, 0, 19, 25881, 100, 0, 0, 0, 0, 0, 0, 'Kaw - Timed - Mount to Moria'),
+(2580200, 9, 4, 0, 0, 0, 100, 0, 0, 0, 0, 0, 0, 19, 33024, 0, 0, 0, 0, 0, 19, 25881, 100, 0, 0, 0, 0, 0, 0, 'Kaw - Remove Moria Flags Immune To Players');
+-- Make her NON_ATTACKABLE until the event starts
+UPDATE `creature_template` SET `unit_flags` = 0 WHERE (`entry` = 25881);
```

Comment:
On Respawn -> Set UnitFlags 768
If only Immune To Player then pets will attack the mammoth as well

Reply:
@Gultask  I'm fixing it here, but this is the closest thing to being a solution for this quest. I did this based on a correction they were making and stopped. I'm already updating SQL :smile: